TDP/tdp-45.md

```diff
@@ -1,7 +1,7 @@
 ---
 tdp: 45
 title: mStableDAO Restructure - Election Process
-status: WIP
+status: Approved

```

Comment:
Approved?

Reply:
Oops! I meant to do that for TDP 44 (so many TDPs :()